### PR TITLE
feat: allowed ccv,globalfee stargate queries #NTRN-408

### PR DESCRIPTION
**Pull request by swelf19**
PR includes:
1) Adding new Stargate requests to the whitelist.
2) Correcting typos in existing requests.
3) Changing the Hermes configuration. This is done to align the local Hermes configuration with the version that has long been used in integration tests (https://github.com/neutron-org/neutron-integration-tests/blob/main/setup/hermes/create-conn.sh#L13). This configuration is used only when manually launching chains using the commands 'make init && make start-rly'.

Task definition: Add updateParamsPersimmon to chain manager (https://github.com/neutron-org/neutron-dao/tree/main/contracts/dao/neutron-chain-manager) for the following modules:
- consumer
- dynamicfees
- globalfee

All prs:
https://github.com/neutron-org/neutronjs/pull/10
https://github.com/neutron-org/neutronjsplus/pull/69
https://github.com/neutron-org/neutron-integration-tests/pull/368
https://github.com/neutron-org/neutron-dao/pull/116
https://github.com/neutron-org/neutron-std/pull/8
https://github.com/neutron-org/neutron/pull/765

### CODE REVIEW

File: network/hermes/create-conn.sh [hunk 1]
```diff
@@ -8,5 +8,14 @@ set -e
 echo "Initiating connection handshake..."
 $HERMES_BINARY --config $CONFIG_DIR create connection --a-chain test-1 --b-chain test-2
 
-sleep 2
-$HERMES_BINARY --config $CONFIG_DIR create channel --a-chain test-1 --a-connection connection-0 --a-port transfer --b-port transfer
+
+
+# a short living client is needed for client update proposal test
+echo "Initiating short living client..."
```
Comment (pr0n00gler) [inline network/hermes/create-conn.sh:14]: Is it possible to move it into the integration tests?
Cause this seems unnecessary in Neutron itself

Comment (mmikev) [inline network/hermes/create-conn.sh]: Is it possible to move it into the integration tests?
Cause this seems unnecessary in Neutron itself

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

**Review by joldie777 — DISMISSED** [review]

File: network/hermes/create-conn.sh (hunk 1 above)
Comment (sotnikov-s) [reply network/hermes/create-conn.sh]: 💯

File: wasmbinding/stargate_allowlist.go [hunk 2]
```diff
@@ -111,12 +113,18 @@ func AcceptedStargateQueries() wasmkeeper.AcceptedQueries {
...
 
 		// feemarket
-		"feemarket.feemarket.v1.Query/Params":    &feemarkettypes.ParamsResponse{},
-		"feemarket.feemarket.v1.Query/State":     &feemarkettypes.StateResponse{},
-		"feemarket.feemarket.v1.Query/GasPrice":  &feemarkettypes.GasPriceResponse{},
-		"feemarket.feemarket.v1.Query/GasPrices": &feemarkettypes.GasPricesResponse{},
+		"/feemarket.feemarket.v1.Query/Params":    &feemarkettypes.ParamsResponse{},
+		"/feemarket.feemarket.v1.Query/State":     &feemarkettypes.StateResponse{},
+		"/feemarket.feemarket.v1.Query/GasPrice":  &feemarkettypes.GasPriceResponse{},
+		"/feemarket.feemarket.v1.Query/GasPrices": &feemarkettypes.GasPricesResponse{},
```
Comment (sotnikov-s) [inline wasmbinding/stargate_allowlist.go:119]: were all these wrong before? (without leading slash)

Comment (swelf19) [reply wasmbinding/stargate_allowlist.go:119]: yes

File: network/hermes/create-conn.sh (hunk 1 above)
Comment (swelf19) [reply network/hermes/create-conn.sh]: We have two sets of configurations for Hermes:
1) For use with the cosmospark in integration tests: https://github.com/neutron-org/neutron-integration-tests/blob/main/setup/hermes/create-conn.sh#L13
2) For local relay run using the commands 'make init && make start-rly'.

I standardized them, as the command has been present in the hermes (docker) configuration for quite some time.

**Review by NeverHappened — DISMISSED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Review by joldie777 — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**